frontend/src/store/NotificationState.ts

```diff
@@ -3,15 +3,27 @@
     It is structured this way so it doesn't stack multiple messages with the same severity (by design).
     It is intended to be used in conjunction with the NotificationStack component.
 */
+import { reactive, ref } from "vue"
 import type { Severity } from "@/enum/SeverityEnum";
-import { ref } from "vue"
 
 const defaultNotification = {
     success: "",
     warn: "",
     error: "",
 };
 
+export const seeAll = reactive({
```

Comment:
Hi Jota, I got what you're trying to do here and I think it's a good idea to store the full message. But I wonder if we could combine the two variables "notifications" and "seeAll" together? like following? otherwise I feel it's a little bit hard to follow and understand, we should always try to keep the logic simple and reusable. Please let me know if you have any questions, thanks!!
```
const defaultNotifications = {
    success: {msg: "", fullMsg: "", showFullMsg: false},
    warn:  {msg: "", fullMsg: "", showFullMsg: false},
    error:  {msg: "", fullMsg: "", showFullMsg: false},
};

export const notifications = ref(
    JSON.parse(JSON.stringify(defaultNotification))
);

export const showFullNotificationMsg = (severity: Severity) => {
    notifications.value[severity].msg = notifications.value[severity].fullMsg
    notifications.value[severity].showFullMsg = true
}
```

Reply:
Hello, Catherine! I can do that